README.md

```diff
@@ -322,7 +332,8 @@ that only the best alignments of each read are kept (smallest edit distance).
 Finally, alignments are intersected with the user-provided, pre-processed
 miRNA annotation file using [BEDTools][bedtools]. Counts are tabulated 
 separately for reads consistent with either miRNA precursors, mature miRNA
-and/or isomiRs.
+and/or isomiRs. Finally, if ASCII-style alignment pileups are generated across
+samples, for the whole run, and across conditions if specified.
```

Comment:
Now you have "Finally" twice :upside_down_face:

Also, the new sentence is incomplete. If x, then ... what? Maybe rather something like:

Finally, ASCII-style alignment pileups are optionally generated for user-defined regions of interest.

Done. But remove the first "Finally" ;-)